authors/settings.py

```diff
@@ -20,7 +20,8 @@
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('SECRET_KEY')
+#SECRET_KEY = os.getenv('SECRET_KEY')
+SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
```

Comment:
@Mutuba it's not safe to leave the secret this vulnerable. he previous implementation was better 
`SECRET_KEY = os.getenv('SECRET_KEY')`